lib/rubocop/cop/rspec/hooks_before_examples.rb

```diff
@@ -30,8 +30,8 @@ class HooksBeforeExamples < Base
 
         def_node_matcher :example_or_group?, <<-PATTERN
           {
-            #{(Examples::ALL + ExampleGroups::ALL).block_pattern}
-            #{Includes::EXAMPLES.send_pattern}
+            #{block_pattern('{#ExampleGroups.all #Examples.all}')}
+            #{send_pattern('#Includes.example')}
```

Comment:
I also just now noticed that `Include::EXAMPLES` was changed to `Includes.example`. I wonder if the plural form was better.